tests/integration/component.go

```diff
@@ -309,4 +309,93 @@ func componentTests(args ...string) {
 		})
 	})
 
+	Context("odo component updating", func() {
+		JustBeforeEach(func() {
+			project = helper.CreateRandProject()
+			context = helper.CreateNewContext()
+		})
+
+		JustAfterEach(func() {
+			helper.DeleteProject(project)
+			os.RemoveAll(context)
+		})
+
+		It("should be able to update a component from local to git", func() {
```

Comment:
Looks duplicate of scenarion in line no 88 ```It("should be able to create a component with git source", func()``` with some more operation and validation. 

@mik-dass would you please update the available scenario rather writing the new one.